adagrams/game.py

```diff
@@ -1,13 +1,262 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+
+def letter_pool_as_list():
+    """Creates a list of the available letters.
+
+    Args:
+        none
+    
+    Returns:
+        list: letter_pool_list (list of strings). Each string is a single capital letter. The number of each capital letter depends on fluency as defined in LETTER_POOL.
+    """
+    letter_pool_list = []
+    for letter, frequency in LETTER_POOL.items():
+        for i in range(0, frequency):
```

Comment:
Love that you are ensuring that the correct weights per letter are being accounted for with this loop. When we are using a for range loop that doesn't utilizes the loop variable, `i`, we can instead use `_` to signal to other developers and ourselves that this loop is just to run our iteration logic **x** amount of times. 
```suggestion
        for _ in range(0, frequency):
```